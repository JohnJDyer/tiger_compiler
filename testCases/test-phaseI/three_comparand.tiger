let
  var a : int := 42;
in
  a := 1=2=3;
end
