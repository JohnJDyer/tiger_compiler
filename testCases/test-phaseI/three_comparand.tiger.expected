	assign, a, 42, 
main:
	assign, $t0, 1, 
	breq, 1, 2, compare_label0
	assign, $t0, 0, 
compare_label0:
	assign, a, $t0, 
	return, , , 
